crates/ui/src/components/tab_bar.rs

```diff
@@ -1,11 +1,10 @@
-use gpui::{AnyElement, ScrollHandle};
-use smallvec::SmallVec;
-
 use crate::prelude::*;
+use gpui::{AnyElement, ScrollHandle, Stateful};
```

Comment:
Let's keep these imports where they were.

We want to separate third-party imports from local `crate::` import blocks.

Reply:
I ran rustfmt and it did that, but you're right - I'll revert this to the previous order.